app/controllers/subscription.js

```diff
@@ -201,6 +202,39 @@ export default Ember.Controller.extend({
         this.updateCartAvailability(0, cartItem);
       }
     }
+
+    // mark message read here
+    if (type === "message") {
```

Comment:
what's the alternative to `message` ? 🤔

Reply:
there are many, order, item, version etc.